Description: Build __munmap_and_exit on x32
 __munmap_and_exit is required by libpthread, so build it. (The x86_64
 variant thereof is compatible with x32, so just reference it from
 Makefile.add.)
Author: Christian Seiler <christian@iwakd.de>
Last-Update: 2017-01-05

--- a/x32/Makefile.add
+++ b/x32/Makefile.add
@@ -1,3 +1,3 @@
 CFLAGS := -Os -fstrict-aliasing -momit-leaf-frame-pointer -mfancy-math-387
 VPATH  := x32:x86_64:syscalls.s:$(VPATH)
-LIBOBJ += $(OBJDIR)/arch_prctl.o $(OBJDIR)/__CAS.o
+LIBOBJ += $(OBJDIR)/arch_prctl.o $(OBJDIR)/__CAS.o $(OBJDIR)/__munmap_and_exit.o
